CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * [#2540](https://github.com/ruby-grape/grape/pull/2540): Introduce Params builder with symbolized short name - [@ericproulx](https://github.com/ericproulx).
 * [#2550](https://github.com/ruby-grape/grape/pull/2550): Drop ActiveSupport 6.0 - [@ericproulx](https://github.com/ericproulx).
 * [#2549](https://github.com/ruby-grape/grape/pull/2549): Delegate cookies management to `Grape::Request` - [@ericproulx](https://github.com/ericproulx).
+* [#2554](https://github.com/ruby-grape/grape/pull/2554): Remove Grape::Http::Headers - [@ericproulx](https://github.com/ericproulx).
```

Comment:
I think you have a fix here for https://github.com/ruby-grape/grape/issues/2334, add?

Reply:
Well its a fix .. but not really. We've been using `Grape::Util::Header` for a while and since then, Grape is compliant with Rack's spec about headers.